### ISSUE TITLE
`statusBarColorControl` option is using 100% of CPU

### ISSUE THREAD

**Issue by ZackMFleischman** [VSCodeVim/Vim#2166]
* Click *thumbs-up* 👍 on this issue if you want it!
* Click *confused* 😕 on this issue if not having it makes VSCodeVim unusable. 

The VSCodeVim team prioritizes issues based on reaction count. 

--------

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): BUG REPORT

**Environment**:

- **VSCode Version**: 1.18.1
- **VsCodeVim Version**: 0.10.4
- **OS**: macOS High Sierra (v10.13.1)

**What happened**:
I noticed the "Code Helper" process was using >100% of the CPU time on one of my cores pretty much constantly, even when I have just restarted VS Code. 

<img width="781" alt="activity_monitor__all_processes__hierarchically_" src="https://user-images.githubusercontent.com/4298857/33095965-43ff633c-ceba-11e7-9fdc-ef7b4334be04.png">

Investigating the process via `ps aux` yielded the process:
```
/Applications/Visual Studio Code.app/Contents/Frameworks/Code Helper.app/Contents/MacOS/Code Helper --type=renderer --js-flags=--nolazy --no-sandbox --primordial-pipe-token=E87E057E02E8101D86E3E0276144C456 --lang=en-US --app-path=/Applications/Visual Studio Code.app/Contents/Resources/app --node-integration=true --webview-tag=true --no-sandbox --background-color=#171717 --disable-blink-features=Auxclick --enable-pinch --num-raster-threads=4 --enable-zero-copy --enable-gpu-memory-buffer-compositor-resources --enable-main-frame-before-activation --content-image-texture-target=0,0,3553;0,1,3553;0,2,3553;0,3,3553;0,4,3553;0,5,3553;0,6,3553;0,7,3553;0,8,3553;0,9,3553;0,10,34037;0,11,34037;0,12,34037;0,13,3553;0,14,3553;0,15,3553;1,0,3553;1,1,3553;1,2,3553;1,3,3553;1,4,3553;1,5,3553;1,6,3553;1,7,3553;1,8,3553;1,9,3553;1,10,34037;1,11,34037;1,12,34037;1,13,3553;1,14,3553;1,15,3553;2,0,3553;2,1,3553;2,2,3553;2,3,3553;2,4,3553;2,5,3553;2,6,3553;2,7,3553;2,8,3553;2,9,3553;2,10,34037;2,11,34037;2,12,34037;2,13,3553;2,14,3553;2,15,3553;3,0,3553;3,1,3553;3,2,3553;3,3,3553;3,4,3553;3,5,34037;3,6,3553;3,7,3553;3,8,3553;3,9,3553;3,10,3553;3,11,3553;3,12,34037;3,13,3553;3,14,34037;3,15,34037;4,0,3553;4,1,3553;4,2,3553;4,3,3553;4,4,3553;4,5,34037;4,6,3553;4,7,3553;4,8,3553;4,9,3553;4,10,3553;4,11,3553;4,12,34037;4,13,3553;4,14,34037;4,15,34037 --service-request-channel-token=E87E057E02E8101D86E3E0276144C456 --renderer-client-id=11
```
Although the part that stood out to me was the flag `--type=renderer`

At this point I was a bit stumped, but I decided to do some sanity checking and turned off all my Extensions, which brought everything to normal CPU usage levels. 

Long story short I narrowed in on the settings:
```
    "vim.statusBarColorControl": true,
    "vim.statusBarColors": {
        "normal": "#005f5f",
        "insert": "#5C0081",
        "visual": "#007022",
        "visualline": "#007022",
        "visualblock": "#86592d",
        "replace": "#000000"
    },
```
and setting `statusBarColorControl` to false fixed the issue. 

**What did you expect to happen**: 
I'd expect `statusBarColorControl` to not have appreciable effect on the CPU usage of the machine.

**How to reproduce it**:
I imagine you can try setting those color options on your machine and see if it cranks up the CPU usage.

**Comment by westim** [comment]
Thanks for the feedback! I believe this is a duplicate of issue #2124.

**Comment by jpoon** [comment]
Closing as a dupe. We've already made note of this issue on our Readme (https://github.com/VSCodeVim/Vim/blob/master/README.md#vimstatusbarcolorcontrol).
